index.html

```diff
@@ -1,216 +1,479 @@
+        この言葉には、「次の一歩をふみ出すきっかけとなるような“ひらめき”や“刺激”を届けたい」という思いが込められています。プログラミングやテクノロジーの世界は、生成AIをはじめ新しいことが次々に生まれ、進化し続けています。けれども、子どもたちの「やってみたい！」「知りたい！」という好奇心は、いつの時代も変わりません。
+      </p>
+
+      <p>
+        CoderDojo は、そんな好奇心を大切にし、支援し、応援する場所です。<br />
+        このイベントを通して、子どもも大人も、次のチャレンジに向かう気持ちに火がつくような出会いや体験を届けたいと考えています。新しい道場を立ち上げてみたくなったり、今の道場をもっと良くしたくなったり――<br />
+        そんな「次」への一歩を、みんなで応援します。
+      </p>
     </div>
-  </body>
+  </div>
+
+  <div class="flex items-center -mx-4 mt-18 justify-between w-full">
+    <div class="w-2/3 relative">
+      <!-- 写真にかぶっても視認性を確保するためのオーバーレイ -->
+      <div class="bg-linear-to-r from-transparent from-70% to-white hidden top-0 xl:block absolute size-full">
+      </div>
+      <img class="w-full" src="/top/img/exhibit.webp" />
+    </div>
+    <div class="w-1/4 -mt-48"><img class="w-full" src="/top/img/fire-transparent.webp" /></div>
+  </div>
+</div>
+
+<!-- モバイルの場合 -->
+<div class="xl:hidden relative mt-20 mx-6">
+  <div class="max-w-3xl mx-auto">
+    <h2 class="relative max-w-7xl">
+      <img src="/top/img/inspire-next-heading.webp" alt="Inspire Next ～好奇心に火をつけよう～" />
+      <img class="h-[140%] w-auto absolute right-0 -bottom-1/5 w-3xs" src="/top/img/fire-transparent.webp" />
+    </h2>
+
+    <div class="mt-9 leading-7">
+      <p class="mb-6">
+        今年の DojoCon Japan のテーマは「Inspire Next 〜好奇心に火をつけよう〜」です。<br />
+        この言葉には、「次の一歩をふみ出すきっかけとなるような“ひらめき”や“刺激”を届けたい」という思いが込められています。プログラミングやテクノロジーの世界は、生成AIをはじめ新しいことが次々に生まれ、進化し続けています。けれども、子どもたちの「やってみたい！」「知りたい！」という好奇心は、いつの時代も変わりません。
+      </p>
+
+      <p>
+        CoderDojo は、そんな好奇心を大切にし、支援し、応援する場所です。<br />
+        このイベントを通して、子どもも大人も、次のチャレンジに向かう気持ちに火がつくような出会いや体験を届けたいと考えています。新しい道場を立ち上げてみたくなったり、今の道場をもっと良くしたくなったり――<br />
+        そんな「次」への一歩を、みんなで応援します。
+      </p>
+    </div>
+  </div>
+
+  <div class="max-w-3xl mx-auto mt-8">
+    <img class="place-self-center w-full" src="/top/img/exhibit.webp" />
+  </div>
+</div>
+
+<h2 class="text-4xl mt-20 sm:mt-32 xl:mt-60 2xl:mt-40 text-center">
+  お知らせ
+  <span class="block mt-5 text-2xl">NEWS</span>
+</h2>
+
+<div class="m-6 flex flex-col items-center">
+  <ul class="border-y border-[#ccc] divide-y divide-[#ccc] max-w-7xl mx-auto mb-12">
+    <li class="text-sm flex items-center py-4 flex-wrap sm:flex-nowrap gap-y-1">
+      <time class="block min-w-24">2025.08.02</time>
+      <span class="block min-w-24 text-center mr-6 px-4 py-px rounded-xs bg-[#cc8f2e] text-white">お知らせ</span>
+      <a href="/contests/how-to-apply.html" class="ml-2 hover:underline">DojoCon Japan 2025 プログラミングコンテスト 応募方法を公開しました</a>
+    </li>
+    <li class="text-sm flex items-center py-4 flex-wrap sm:flex-nowrap gap-y-1">
+      <time class="block min-w-24">2025.08.02</time>
+      <span class="block min-w-24 text-center mr-6 px-4 py-px rounded-xs bg-[#cc8f2e] text-white">お知らせ</span>
+      <a href="/sponsorship/index.html" class="ml-2 hover:underline">DojoCon 2025 スポンサー募集のお知らせを掲載しました</a>
```

Comment:
- （ティザーの時点で誤りがあるが）"DojoCon Japan 2025" が正しい
- 遷移先の記事タイトルは、正しく "DojoCon Japan 2025" になっている
```suggestion
      <a href="/sponsorship/index.html" class="ml-2 hover:underline">DojoCon Japan 2025 スポンサー募集のお知らせを掲載しました</a>
```